documentation/modules/deploying/proc-deploying-kafkaconnector.adoc

```diff
@@ -30,27 +43,137 @@ spec:
   class: org.apache.kafka.connect.file.FileStreamSourceConnector <3>
   tasksMax: 2 <4>
   config: <5>
-    file: "/opt/kafka/LICENSE"
-    topic: my-topic
+    file: "/tmp/my-file" <6>
```

Comment:
This one (for the Source) should stay `/opt/kafka/LICENSE`, since its a text file that exists in the container and saves anyone experimenting from having to shell into the container to create their own file.

Reply:
Gah, how didn't I notice that?!